Website/Packages/datax-pipeline/src/modules/flowDefinition/components/schedule/oneTimeScheduleSettings.jsx

```diff
@@ -0,0 +1,234 @@
+                    disabled={true}
+                    value={this.props.batchTypeDisplayName}
+                />
+            </div>
+        );
+    }
+
+    renderTimeRange() {
+        return (
+            <div >
+                {this.renderStartTime()}
+                {this.renderEndTime()}
+            </div>
+        );
+    }
+    
+    renderStartTime() {
+        return (
+            <div style={sectionStyle}>
+                <Label className="ms-font-m info-settings-textbox">Start Time</Label>
+                <DateTimePicker
+                    className="ms-font-m info-settings-textbox"
+                    value={this.props.batch.properties.startTime}
+                    onChange={(value) => this.props.onUpdateBatchStartTime(value)}
+                />
+            </div>
+        );
+    }
+
+    renderEndTime() {
+        return (
+            <div style={sectionStyle}>
+                <Label className="ms-font-m info-settings-textbox">End Time</Label>
+                <DateTimePicker
+                    className="ms-font-m info-settings-textbox"
+                    value={this.props.batch.properties.endTime}
+                    onChange={(value) => this.props.onUpdateBatchEndTime(value)}
+                />
+            </div>
+        );
+    }
+
+    renderInterval() {
+        return (
+            <div style={sectionContainerStyle}>
+                {this.renderIntervalText()}
+                {this.renderIntervalTypeDropdown()}
+            </div>
+        );
+    }
+
+    renderIntervalText() {
+        return (
+            <div style={sectionValueStyle}>
+                <TextField
+                    className="ms-font-m"
+                    spellCheck={false}
+                    label="Interval"
+                    value={this.props.batch.properties.interval}
+                    onChange={(event, value) => this.props.onUpdateBatchIntervalValue(value)}
+                />
+            </div>
+        );
+    }
+
+    renderIntervalTypeDropdown() {
+        const options = Models.batchIntervalTypes.map(type => {
+            return {
+                key: type.key,
+                text: type.name,
+                disabled: type.disabled
+            };
+        });
+
+        return (
+            <div style={sectionDropdownStyle}>
+                <Label className="ms-font-m">Unit</Label>
+                <Dropdown
+                    className="ms-font-m"
+                    options={options}
+                    selectedKey={this.props.batch.properties.intervalType}
+                    onChange={(event, selection) => this.props.onUpdateBatchIntervalType(selection.key)}
+                />
+            </div>
+        );
+    }
+
+    renderWindow() {
+        return (
+            <div style={sectionContainerStyle}>
+                {this.renderWindowText()}
+                {this.renderWindowDropdown()}
+            </div>
+        );
+    }
+
+    renderWindowText() {
+        return (
```

Comment:
prettier would have formatted this correctly on save. you just need to open the datax-pipeline package folder to kick in. See my other comments I added previously.

Reply:
done.